test/grpc/cancelled-by-client.cc

```diff
@@ -23,7 +23,7 @@ using eventuals::grpc::CompletionPool;
 using eventuals::grpc::Server;
 using eventuals::grpc::ServerBuilder;
 
-TEST_F(EventualsGrpcTest, CancelledByClient) {
+TEST(CancelledTest, CancelledByClient) {
```

Comment:
To stay consistent with existing naming based on files can I suggest:
```suggestion
TEST(CancelledByClientTest, Cancelled) {
```

Reply:
Done